src/ompl/tools/thunder/src/SPARSdb.cpp

```diff
@@ -195,22 +199,27 @@ bool ompl::geometric::SPARSdb::getSimilarPaths(int /*nearestK*/, const base::Sta
                                                CandidateSolution &candidateSolution,
                                                const base::PlannerTerminationCondition &ptc)
 {
+    std::ofstream rr_planner_debug;
+    rr_planner_debug.open("zzz_rr_planner_debug.txt", std::ios::app);
```

Comment:
is this needed during normal operation? we can keep the code, but control with a flag

Reply:
These debugging logs were not left in the PR on purpose. I will remove most of them, and the ones that I will feel need to be kept, I will keep with a flag as you suggested.